exodus_lambda/functions/origin_response.py

```diff
@@ -33,6 +33,12 @@ def handler(self, event, context):
                 {"key": "Digest", "value": f"id-sha-256={sum_b64}"}
             ]
 
+        if "headers" in request and "x-exodus-query" in request["headers"]:
+            self.logger.error("x-exodus-query exist")
```

Comment:
Please drop this as it's not an error.  You could keep a debug() if you really need it.